.swiftpm/xcode/xcshareddata/xcschemes/ReceiptParser.xcscheme

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```

Comment:
these are the changes from https://github.com/RevenueCat/purchases-ios/pull/2776, right? do we need to rebase?

Reply:
Done.